packages/sarif-to-comment/src/cli.ts

```diff
@@ -1,7 +1,7 @@
 import meow from "meow";
 import { postComment } from "./index";
 import * as fs from "fs";
-
+const ALLOWED_SEVERITIES = ["warning", "error", "note", "none"];
```

Comment:
```suggestion
const ALLOWED_SEVERITIES = ["warning", "error", "note", "none"] as const;
```

To use const assertions make more safe.
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions